xla/debug_options_flags.cc

```diff
@@ -1428,6 +1428,13 @@ void MakeDebugOptionsFlags(std::vector<tsl::Flag>* flag_list,
       bool_setter_for(&DebugOptions::set_xla_gpu_enable_cub_radix_sort),
       debug_options->xla_gpu_enable_cub_radix_sort(),
       "Enable radix sort using CUB for simple shapes"));
+  flag_list->push_back(tsl::Flag(
+      "xla_gpu_threshold_for_windowed_einsum_mib",
+      int64_setter_for(
+          &DebugOptions::set_xla_gpu_threshold_for_windowed_einsum_mib),
+      debug_options->xla_gpu_threshold_for_windowed_einsum_mib(),
+      "Threshold to enable windowed einsum (collective matmul) in MB."
+      "Default is 100000"));
```

Comment:
I always find that kind of comment unclear.
Is collective_matmul enabled when one input tensor is above that size? When the sum of both is above that size? Under that size? Can you clarify this here?

Reply:
sorry i was OOO for the last 2 weeks and missed this comment. This is actually for just lhs or rhs of the dot depending on the sharding strategy. I will make this more descriptive in the next pr.